apps/demo/app/routes/index.tsx

```diff
@@ -95,9 +101,9 @@ export default function Index() {
             header: 'Header',
             headingLevel: 'h2',
             src: 'https://i.ibb.co/MS0V7Zk/elisa-pitkanen-Z9-FZ6hko-k-unsplash.jpg',
-            ['data-radius-watch-token-changes']: true,
-            ['data-radius-token']:
-              '--asset-component-image-large-image-text-item-image01',
+            // ['data-radius-watch-token-changes']: true,
+            // ['data-radius-token']:
+            //   '--asset-component-image-large-image-text-item-image01',
```

Comment:
Are these needed?

Reply:
I don't think so no. Seems like a debugging artifact

Reply:
Actually, this might be what @jasonsantos  was using to watch for changes to context?